packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -489,6 +491,58 @@ export default abstract class SqlIntegration
     });
   }
 
+  formatInformationSchema(
+    results: RawInformationSchema[],
+    datasourceType: string
+  ): FormattedInformationSchema[] {
```

Comment:
```suggestion
  ): InformationSchema[] {
```
From a naming perspective, I would suggest calling it `InformationSchema` instead, as in "this is a GrowthBook InformationSchema" which is our standardized data model to represent any datasource's IS.

Contrast with `FormattedInformationSchema`, which implies there is non-formatted information schema somewhere ... My inclination is to avoid that because the 'raw' or 'non-formatted' IS should be handled as near to the datasource as possible, so that within Growthbook application space, we only deal with `InformationSchema` objects

Reply:
Solid logic - I'll get this updated!